values_test.go

```diff
@@ -329,24 +329,24 @@ func TestToBigEndianBytes(t *testing.T) {
 			NewInt64(-9223372036854775808): {128, 0, 0, 0, 0, 0, 0, 0},
 		},
 		"Int128": {
-			NewInt128(0):                  {0},
-			NewInt128(42):                 {42},
-			NewInt128(127):                {127},
-			NewInt128(128):                {0, 128},
-			NewInt128(200):                {0, 200},
-			NewInt128(-1):                 {255},
-			NewInt128(-200):               {255, 56},
-			NewInt128(-10000000000000000): {220, 121, 13, 144, 63, 0, 0},
+			NewInt128(0):                  {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+			NewInt128(42):                 {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42},
+			NewInt128(127):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 127},
+			NewInt128(128):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 128},
+			NewInt128(200):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 200},
+			NewInt128(-1):                 {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255},
+			NewInt128(-200):               {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 56},
+			NewInt128(-10000000000000000): {255, 255, 255, 255, 255, 255, 255, 255, 255, 220, 121, 13, 144, 63, 0, 0},
 		},
 		"Int256": {
-			NewInt256(0):                  {0},
-			NewInt256(42):                 {42},
-			NewInt256(127):                {127},
-			NewInt256(128):                {0, 128},
-			NewInt256(200):                {0, 200},
-			NewInt256(-1):                 {255},
-			NewInt256(-200):               {255, 56},
-			NewInt256(-10000000000000000): {220, 121, 13, 144, 63, 0, 0},
+			NewInt256(0):                  {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+			NewInt256(42):                 {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42},
+			NewInt256(127):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 127},
+			NewInt256(128):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 128},
+			NewInt256(200):                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 200},
+			NewInt256(-1):                 {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255},
+			NewInt256(-200):               {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 56},
+			NewInt256(-10000000000000000): {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 220, 121, 13, 144, 63, 0, 0},
```

Comment:
didn't add the min/max cases to this test suite because we can't construct an Int256 by passing in a machine int.

Reply:
You can construct those values with `Int256{...}`. `sema` defines min/max values for all types, e.g. you can use `Int256TypeMaxIntBig`